src/Approvals/SFA.DAS.Approvals/Services/ProviderCoursesService.cs

```diff
@@ -98,12 +104,26 @@ private async Task<IEnumerable<Standard>> GetAllStandards()
 
         private async Task<IEnumerable<Standard>> GetStandardsForProvider(long providerId)
         {
-            var providerStandards =
-                await _providerCoursesApiClient.Get<IEnumerable<GetProviderStandardsResponse>>(
-                    new GetProviderStandardsRequest(providerId));
+            try
+            {
+                var providerStandards =
+                    await _providerCoursesApiClient.Get<IEnumerable<GetProviderStandardsResponse>>(
+                        new GetProviderStandardsRequest(providerId));
+
+                if (providerStandards.IsNullOrEmpty())
+                {
+                    _logger.LogWarning("No Standards Declared");
+                    return null;
+                }
 
-            return providerStandards.Select(
+                return providerStandards.Select(
                 x => new Standard(x.LarsCode.ToString(), x.CourseNameWithLevel)).OrderBy(x => x.Name);
+            }
+            catch (Exception e)
+            {
+                _logger.LogError("No Standards Declared", e);
+                return null;
```

Comment:
Why return null? We could simply return an empty enumerable couldn't we, which would indicate the same thing?

Reply:
Done